client/routes/search.js

```diff
@@ -45,6 +45,10 @@ module.exports = function (page) {
       var pageEl = document.getElementsByTagName('main')[0];
       pageEl.innerHTML = Templates['search'](ctx.state.data);
 
+      // Shows filter toggle button if javascript enabled
+      document.getElementById('fb').className = 'control__button';
```

Comment:
is the objective to _overwrite_ the class on this element or add an additional one? 
e.g. should it be `document.getElementById('fb').className += 'control__button';`
see: http://stackoverflow.com/questions/13980982/add-class-to-html-with-javascript